packages/shared-utils/configs/networks.json

```diff
@@ -0,0 +1,439 @@
+[
```

Comment:
Do you think it's necessary to keep this file? Personally, I would suggest deleting it and only retaining https://github.com/forbole/big-dipper-networks/blob/main/networks.json. What is your opinion on this?

Reply:
yes I think we can remove this if there's no reference to this file, instead we can move this generated file to update the `big-dipper-networks` repo